css/index.css

```diff
@@ -0,0 +1,168 @@
+.side-navbar {
+  background-color: rgb(25 146 212);
+  width: 200px;
+}
+
+.menuitem {
+  display: flex;
+}
+
+img {
+  width: 20px;
+  padding-left: 15px;
+}
+
+h1 {
+  font-size: 20px;
+  padding-left: 25px;
+}
+
+.container {
+  display: flex;
+}
+
+.content {
+  background-color: rgb(226 232 240);
+  width: 100%;
+  height: 700px;
+  padding-bottom: 100px;
+}
+
+.top-navbar {
+  display: flex;
+  padding-top: 20px;
+}
+
+input {
+  width: 550px;
+  border-radius: 10px;
+  background-color: rgb(226 232 240);
+  border: 2px solid rgb(226 232 240);
+  margin-left: 20px;
+}
+
+.navbar {
+  background-color: #fff;
+  height: 130px;
+}
+
+.account {
+  margin-left: 50px;
+}
+
+.user-name {
+  margin-left: 20px;
+}
+
+.bottom-navbar {
+  display: flex;
+}
+
+.g-avatar {
+  display: flex;
+}
+
+.buttons {
+  margin-top: 40px;
+  margin-left: 250px;
+  border-radius: 10px;
+}
+
+.btn {
+  margin-left: 20px;
+  width: 90px;
+  background-color: rgb(25 146 212);
+  border: 2px solid rgb(25 146 212);
+  border-radius: 10px;
+}
+
+.profile {
+  height: 70px;
+  margin-top: 10px;
+  display: flex;
+}
+
+.card {
+  margin-left: 15px;
+  background-color: #fff;
+  width: 350px;
+  height: 180px;
+  margin-bottom: 20px;
+  padding-left: 10px;
+  border-left: 7px solid #f59e0b;
+  margin-right: 5px;
+}
+
+.section1,
+.section2,
+.section3 {
+  display: flex;
+}
+
+.icon {
+  margin-top: 50px;
+  margin-left: 200px;
+}
+
+.sub-section {
+  display: flex;
+}
+
+.box {
+  width: 235px;
+  height: 270px;
+  background-color: #fff;
+  margin-left: 15px;
+  padding-left: 20px;
+  font-size: 12px;
+  padding-bottom: 5px;
+  padding-top: 1px;
+  margin-top: 15px;
+}
+
+.side-container {
+  margin-bottom: 10px;
+  margin-top: 5px;
+}
+
+.box-1 {
+  width: 235px;
+  height: 255px;
+  background-color: #fff;
+  margin-left: 15px;
+  padding-left: 20px;
+  font-size: 12px;
+  padding-bottom: 0;
+}
+
+.projects {
+  margin-top: 5px;
+}
+
+h2 {
+  font-size: 18px;
+  padding-left: 25px;
+}
+
+.card details {
+  margin-bottom: 18px;
+}
+
+.details {
+  margin-bottom: -3%;
+}
+
+h3 {
```

Comment:
@Gbemisola19 Here too, let give all those elements class name, making use of class name helps provide consistency and reusability